src/main/java/com/moodybluez/enterprise/dao/IMoodDAO.java

```diff
@@ -0,0 +1,16 @@
+package com.moodybluez.enterprise.dao;
+
+import com.moodybluez.enterprise.dto.Mood;
+
+import java.util.Map;
+
+public interface IMoodDAO {
+
+    boolean createEntry(Mood mood);
+
+    Mood fetchByMood(String mood);
+
+    Mood fetchByMoodID(int moodID);
+
+    Map<Integer, Mood> fetchAll();
```

Comment:
Added Persistence level methods (fully implemented with logic in actual classes) to this layer, moved from service